### ISSUE TITLE
2e ronde met beste nr's x

### ISSUE THREAD

**Issue by brasei** [thepercival/fctoernooi#74]
Hallo,
Graag zouden we een toernooi opzetten met 4 poules van bv. 5 teams. 
De 2e ronde moet vervolgens bestaan uit weer 4 poules van de nummers 1 + de beste nr 2, de rest van de nrs. 2 + de beste nrs 3, enz.
Het lukt nu bijna alleen plaatst hij de beste nummers in de tweede poule en voegt ze niet toe aan de eerste:
![image](https://user-images.githubusercontent.com/34542752/208998172-9a352c21-9c93-4162-ab7d-9850810f2f9a.png)

Zou dit aangepast kunnen worden zodat er weer poules van 5 teams bestaan in de tweede ronde?

**Comment by thepercival** [comment]
Hoi, ik begrijp je probleem. Bedankt voor je melding en de heldere uitleg. Je wens is nu niet uitvoerbaar met fctoernooi. Je zou 5 poules kunnen doen, met in elke poule dezelfde rankings uit de verschillende poules tegen elkaar. Dit wil je waarschijnlijk niet, omdat je hier een andere wens hebt aangegeven. :-) 

Deze vraag hebben we nog niet eerder gehad. Op korte termijn is dit niet uitvoerbaar, maar ik wil wel graag deze melding laten staan om te kijken als we deze functionaliteit toch kunnen toevoegen. Ik wil ook onderzoeken als je wens breder toepasbaar is binnen fctoernooi. De structuur aanmaken op een flexibele manier is erg ingewikkeld en kost dus ook tijd om goed te implementeren.

**Comment by brasei** [comment]
Oké helaas, het leek me te kunnen omdat hij het dus wel doet in die tweede poule, maar hij schuift ze dus niet door naar de eerste poule als er meer teams in de tweede poule komen dan in de eerste.
Zou dat nog een richting kunnen zijn? Aantal teams in poule kruisfinale(2e ronde) = totaal winnaars / aantal winnaarspoules?

Bedankt iig voor je snelle antwoord!

**Comment by thepercival** [comment]
Klopt, er is nu alleen geen mogelijkheid dat er doorgenummerd wordt. Ik laat de melding open om te kijken als we deze optie kunnen toevoegen, maar ik wil de tool niet te ingewikkeld maken. Bedankt voor de inbreng.

**Comment by thepercival** [comment]
Dit is inmiddels mogelijk. Je kan het gebruiken door de "verticaal" kwalificeren aan te zetten. 


![Screenshot_20240426-051241.png](https://github.com/thepercival/fctoernooi/assets/1693544/cebb708b-acc4-4938-a50e-7684c0677cc0)


Het daarna separaat doorwerken kan nog niet. Dit is opgenomen in
https://github.com/thepercival/fctoernooi/issues/87
